src/frontc/cprint.ml

```diff
@@ -538,6 +540,21 @@ and print_expression_level (lvl: int) (exp : expression) =
       printl ["__alignof__";"("];
       print_onlytype (bt, dt);
       print ")"
+  | GENERIC (exp, lst) -> 
+      (* TODO: is this right? *)
```

Comment:
why does the `print_generic_list` only print its first argument? That is a bit odd. If it is correct, there should be a comment why it is.

Reply:
You're right, it probably is wrong. I made it recurse to print all the bindings although I haven't tested this Cprint thing at all (nor has anyone else?). The operator precedences here don'y all even match with the ones in `get_operator` I think.